src/components/Nav.module.css

```diff
@@ -8,8 +8,14 @@
 
```

Comment:
- [ ] Please kindly use the same color and font as it used in the original design.

### Your Design
![image](https://user-images.githubusercontent.com/43995830/179131999-fea07b2e-d95a-46df-a620-f6b3cd29d90c.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/179131920-d6023a5e-0e9a-45bd-8d41-e7feaaf575c5.png)

Reply:
@AnselemOdims thank you, I have fixed this up.